### PR TITLE
Update ansible tmp path at cfg file

### PR DESCRIPTION
**Pull request by slutati1536**
### Description of PR

Summary:

update the tmp path at ansible cfg file, so that user other than root could have permissions to tmp folder

### Type of change

- [ ] Bug fix
- [x] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 201911
- [x] 202012
- [x] 202205

#### What is the motivation for this PR?
It is necessary for Ansible scripts to work for users other than root.
As some servers don't use root access, other users should have permission for /tmp folder.

#### How did you do it?
change the cfg file

#### How did you verify/test it?
test it by running full community regression

#### Any platform-specific information?
no

### CODE REVIEW

File: ansible/ansible.cfg [hunk 1]
```diff
@@ -14,7 +14,7 @@
 inventory      = /etc/ansible/hosts
 library        = library:library/ixia
 module_utils   = module_utils
-remote_tmp     = $HOME/.ansible/tmp
+remote_tmp     = /tmp/ansible-$USER
```
Comment (nazariig) [inline ansible/ansible.cfg:17]: @slutati1536 suggest to have: `/tmp/.ansible-$USER`

**Review by nazariig — CHANGES_REQUESTED** [review]

**Comment by slutati1536** [comment]
@roy-sror  Hi Roy, can you please approve the commit?

**Review by roy-sror — APPROVED** [review]

**Comment by liat-grozovik** [comment]
@nazariig  all good?

**Comment by nazariig** [comment]
@slutati1536 please fix the review comments

File: ansible/ansible.cfg (hunk 1 above)
Comment (slutati1536) [reply ansible/ansible.cfg:17]: @nazariig I will test this change in regression tonight, and will update the PR

**Review by nazariig — APPROVED** [review]

**Review by liat-grozovik — APPROVED** [review]

**Comment by liat-grozovik** [comment]
@slutati1536 pls check if the PR can be smoothly cherry picked to the requested branches. if not please create dedicated PR.

**Pull request merged**